build_common.ps1

```diff
@@ -1211,28 +1211,42 @@ class BufferingReceiver : StdoutReceiver {
 
 
 class MakeStdoutReceiver : StdoutReceiver {
-	[string] $developmentDirectory
-	[string] $modSrcRoot
+	[BuildProject] $proj
+	[string[]] $reversePaths
 
 	MakeStdoutReceiver(
-		[string]$developmentDirectory,
-		[string]$modSrcRoot
+		[BuildProject]$proj
 	){
-		$this.developmentDirectory = $developmentDirectory
-		$this.modSrcRoot = $modSrcRoot
+		$this.proj = $proj
+		# Since later paths overwrite earlier files, check paths in reverse order
+		# Also include the Development\Src directory to catch files that weren't included via the proper mechanisms
```

Comment:
Instead of relying on `Development\Src` being present in the lookup, can we instead add a fallback to the loop later (that would use the original path) in case none were matched? I feel like that would be more bulletproof than this approach